website/docs/configuration/providers.html.md

```diff
@@ -172,14 +178,26 @@ configuration. For providers that have no required configuration arguments, the
 implied _empty_ configuration is considered to be the _default_ provider
 configuration.
 
-Resources are normally associated with the default provider configuration
-inferred from the resource type name. For example, a resource of type
-`aws_instance` uses the _default_ (un-aliased) `aws` provider configuration
-unless otherwise stated.
+### Referring to Alternate Providers
```

Comment:
(this, below, was a bit that I remember having trouble with; could use confirmation that I got it right.)